File: HackerHank/Pangrams/PangramsSolution.py
def pangrams(s):
    letters = [None] * 26
    counter = 0
    for i in range(ord('a'), ord('z')+1):
        letters[counter] = i
        counter += 1
    frequency = [0]*26
    containAll = True
    print(frequency)
    for i in range(0, len(s)):
        if(s[i] != ' '):
            frequency[ord(s[i].lower())-ord('a')] += 1
    for i in range(0, 26):
        if(frequency[i] == 0):
            containAll = False
    if(containAll):
        return "pangram"
    return "not pangram"

File: HackerHank/Pangrams/PangramsSolution.cpp
#include <string>
#include <algorithm>
using namespace std;
string pangrams(string s)
{
    std::for_each(s.begin(), s.end(), [](char &i)
                  { i = ::tolower(i); });

    char letters[26];
    int frequency[26];
    bool containAll = 1;
    for (int i = 0; i < 26; i++)
        letters[i] = 'a' + i;
    if (s.size() < 26)
        return "not pangram";
    for (int i = 0; i < 26; i++)
        frequency[i] = 0;
    for (int i = 0; i < s.size(); i++)
        frequency[s[i] - 'a']++;
    for (int i = 0; i < 26; i++)
        if (frequency[i] == 0)
            containAll = 0;
    if (containAll)
        return "pangram";
    return "not pangram";
}